src/analysis/retail/shaman/elemental/modules/Abilities.tsx

```diff
@@ -24,20 +25,20 @@ class Abilities extends CoreAbilities {
         },
       },
       {
-        spell: SPELLS.LIQUID_MAGMA_TOTEM_TALENT.id,
+        spell: SPELLS.LIQUID_MAGMA_TOTEM_DAMAGE.id,
```

Comment:
similar here
```suggestion
        spell: TALENTS.LIQUID_MAGMA_TOTEM_TALENT.id,
```

Reply:
changed to TALENTS.LIQUID_MAGMA_TOTEM_TALENT.id